.codeclimate.yml

```diff
@@ -8,7 +8,8 @@ engines:
     enabled: true
     config:
       languages:
-      - javascript
+        javascript:
```

Comment:
I'd like to merge this PR, but we are not done discussing the change yet in https://github.com/ManageIQ/manageiq-ui-self_service/issues/204